docs/api/types.md

```diff
@@ -171,14 +171,67 @@ type Notification = {
     | "mod_add";
 
   // The content of the notification. The structure of this object will vary based on the type of notification.
-  notif: {};
+  notif:
+    | newVotesNotif
+    | deletedPostNotif
+    | newCommentNotif
+    | commentReplyNotif
+    | newBadgeNotif
+    | modAddNotif;
 
   seen: boolean; // Whether the notification was seen by the authenticated user.
   seenAt: time | null; // If the notification was seen, the time at which it was seen, otherwise null.
   createdAt: time; // The time at which the notification was created.
 };
+
+type newVotesNotif = {
```

Comment:
The type names should be in PascalCase. So `DeletedPostNotif` instead of `deletedPostNotif`.